src/functinalities.js

```diff
@@ -0,0 +1,56 @@
+let mylist = JSON.parse(localStorage.getItem('mylist')) || [];// eslint-disable-line
+
+const listGroup = document.querySelector('.todo-group');
+const newTask = document.querySelector('.todo-add').querySelector('input');
+const getmylist = () => {
+  const mylistElement = mylist.map((item) => `
+        <li class="todo-list todo-item" id=${item.index}>
+          ${item.completed === true ? `
+            <i class="fa-solid fa-check checked-icon"></i>`
+    : '<i class="fa-solid fa-square unchecked-icon"></i>'}
+          <input type="text" class=${item.completed === true ? 'decoration edit-todo' : ' edit-todo'}  value="${item.description}">
+          <span class="edit-focus-element"></span>
+          <i class="fa-solid fa-trash-can delete-icon"></i>
+          <i class="fa-solid fa-ellipsis-vertical more-icon"></i>
+        </li>`).join('');
+  listGroup.innerHTML = mylistElement;
+  return listGroup;
+};
+
+const addmylist = (event) => {
```

Comment:
- [ ] [OPTIONAL] Kindly clear the input after adding a task for a better user experience.